app/posts/find-teacher-training/2023-04-18-removing-school-placement-vacancies.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Removing school placement vacancies
+description: We removed the school placement vacancy status as this was misleading for candidates
+date: 2023-04-18
+tags:
+  - school placements
+  - vacancies
+related:
+  items:
+    - text: Updating course vacancies
+      href: /publish-teacher-training-courses/updating-course-vacancies/
+---
+
+When a provider publishes a course, they can list all the schools where a candidate can gain experience. Previously, this list also showed whether there was a vacancy at the school.
+
+The vacancy status was a simple ‘yes’ or ‘no’ and required the provider to keep the course current.
+
+We removed the vacancy status because:
+
+- candidates do not apply to a vacancy at a school
+- providers often do not know the number of vacancies at their partner schools until late in the recruitment cycle
+- the number of vacancies can change during the recruitment cycle because candidates may need to withdraw
+- we are providing false hope that candidates will be placed at certain schools
+- providers do not always consider a candidate’s school placement preference on the application, and some don’t know about the preference
+- the accuracy of vacancy information depends on providers frequently updating their courses
+- Apply for teacher training (Apply) is going to remove the preference for placement schools
+
+## What we changed
+
+We have changed the:
+
+- school placements section of the course details
+- warning text when a course is closed
+- vacancies filter on the course list
+
+### School placements section
+
+We replaced the ‘Schools’ table with a list of schools and removed each school’s vacancy status.
+
+![School placements](school-placements.png "School placements")
+
+### Warning text
+
+If a course is closed, it is still available on Find, but candidates cannot apply.
+
+For all closed courses, we show a message:
+
+> You cannot apply for this course as it is closed for applications. To find courses accepting applications, change your search to ‘Only show courses accepting applications’.
+
+![Course closed for applications](warning-message.png "Course closed for applications")
+
+### Vacancies filter
+
+If a course is open, it is available on Find, and candidates can apply.
+
+We replaced the ‘Vacancies’ filter with one labelled ‘Applications open’. The filter is selected by default, so we ‘Only show courses accepting applications.’
+
+We show all ‘Open’ and ‘Closed’ courses if the filter is deselected.
+
+![Applications open filter](applications-open-filter.png "Applications open filter")
+
+## Further considerations
+
+If someone deselects the ‘Applications open’ filter, we show all open and closed courses. It is not easy to see which courses accept applications and which do not.
+
+In the future, we could add a label to ‘Closed’ courses on the courses list so that users can quickly see whether they can apply for the course.
```

Comment:
Do we need to consider `Date you can apply from` 

![image](https://user-images.githubusercontent.com/470137/235158499-2b7345ef-60c3-475b-9efb-abce83144a54.png)

Reply:
@defong can you elaborate on what we need to consider? For example, do you think we need to change the information if the course is closed? 

Since `Date you can apply from` corresponds to the `Applications open date` in Publish it is possible candidates might be confused if the course is now closed. /cc @Kateharries

Reply:
Yes this is a good point. Can we hide 'Date you can apply from' for closed courses? Ideally, we'd test this with candidates to see if they'd prefer to have that info there (even if the course is closed) or if it's just confusing to have it there. I think we should hide it if the course is closed.

Reply:
We're going to leave the date where it is and add: '- closed for applications' next to the date for courses that are closed.